src/components/Missions.js

```diff
@@ -0,0 +1,26 @@
+import React, { useEffect } from 'react';
```

Comment:
# status: Approved!

Your milestone is up to the requirements and there is no more else to add. I'll make some comments on the highlights of your project